CryptKeeperZK/crypt-keeper-extension
# feat: download backup ui

## Explanation

This PR is a part of backup feature implementation.

Details are below:
- [x] Add download backup ui page
- [x] Add feature flag for backup
- [x] Add download backup section in advanced settings

Next PRs will bring these things but blocked by #305:
1. Support history for backups
2. Password check when downloading backup
3. Generate file on ui side and download it
4. E2E tests

## More Information

Related to #257 

## Screenshots/Screencaps

![image](https://user-images.githubusercontent.com/14254374/235797967-333c780e-dfa8-4024-91b2-f82f1b2bf6c5.png)
![image](https://user-images.githubusercontent.com/14254374/235806265-e9eded97-e84e-468e-8f4d-deb502707566.png)


## Manual Testing Steps

1. Enable feature `BACKUP`
5. Go to `Settings` and click `Advanced` tab
6. See section related to backup

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
